docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Documentation
```

Comment:
@stijnmoreels this should give you guidance how to add more stuff. is this helpful?

I'd suggest that we include all our blog posts and you give that a go, what do you think?

Reply:
Yes, great! Clear and to the point.